app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -160,33 +157,36 @@ function InPersonLocationStep({ onChange, toPreviousStep }) {
     };
   }, []);
 
-  let locationItems: React.ReactNode;
+  let locationsContent: React.ReactNode;
   if (!isLoadingComplete) {
-    locationItems = <SpinnerDots />;
+    locationsContent = <SpinnerDots />;
   } else if (locationData.length < 1) {
-    locationItems = <h4>{t('in_person_proofing.body.location.none_found')}</h4>;
+    locationsContent = <h4>{t('in_person_proofing.body.location.none_found')}</h4>;
   } else {
-    locationItems = locationData.map((item, index) => (
-      <LocationCollectionItem
-        key={`${index}-${item.name}`}
-        handleSelect={handleLocationSelect}
-        name={`${item.name} — ${t('in_person_proofing.body.location.post_office')}`}
-        streetAddress={item.streetAddress}
-        selectId={item.id}
-        formattedCityStateZip={item.formattedCityStateZip}
-        weekdayHours={item.weekdayHours}
-        saturdayHours={item.saturdayHours}
-        sundayHours={item.sundayHours}
-      />
-    ));
+    locationsContent = (
+      <LocationCollection>
+        {locationData.map((item, index) => (
+          <LocationCollectionItem
+            key={`${index}-${item.name}`}
+            handleSelect={handleLocationSelect}
+            name={`${item.name} — ${t('in_person_proofing.body.location.post_office')}`}
+            streetAddress={item.streetAddress}
+            selectId={item.id}
+            formattedCityStateZip={item.formattedCityStateZip}
+            weekdayHours={item.weekdayHours}
+            saturdayHours={item.saturdayHours}
+            sundayHours={item.sundayHours}
+          />
+        ))}
+      </LocationCollection>
+    );
   }
 
   return (
     <>
       <PageHeading>{t('in_person_proofing.headings.location')}</PageHeading>
-
       <p>{t('in_person_proofing.body.location.location_step_about')}</p>
-      <LocationCollection>{locationItems}</LocationCollection>
+      {locationsContent}
```

Comment:
Change here fixes issue where previously we were rendering invalid content inside the `LocationCollection`'s `<ul>`:

>**Permitted content** | Zero or more `<li>`, `<script>` and `<template>` elements.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul